### PR TITLE
Foodland Scraper w/ MongoDB Integration - testing needed

### PR DESCRIPTION
**Pull request by B-JThomas**

### CODE REVIEW

File: Scrapping/Foodland Scraper/scraper_foodland.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    return 1
+
+# Setup MongoDB connection
+def setup_mongo():
+    # Load environment variables from .env file if it exists
+    load_dotenv()
+    
+    # MongoDB credentials
+    username = os.getenv('MONGO_USERNAME', 'discountmate')
+    password = os.getenv('MONGO_PASSWORD', 'discountmate1')
```
Comment (rashannon-sys) [inline Scrapping/Foodland Scraper/scraper_foodland.py:46]: Please remove the actual username and password

**Review by rashannon-sys — CHANGES_REQUESTED** [review]
please remove the companies username and password to the database

File: Scrapping/Foodland Scraper/scraper_foodland.py (hunk 1 above)
Comment (B-JThomas) [reply Scrapping/Foodland Scraper/scraper_foodland.py]: Regardless I have sent a new version throguh

File: Scrapping/Foodland Scraper/scraper_foodland.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    # MongoDB credentials
+    username = os.getenv('MONGO_USERNAME', 'example')
+    password = os.getenv('MONGO_PASSWORD', 'example1')
+    
+    # Encode credentials for MongoDB URI
+    encoded_username = quote_plus(username)
+    encoded_password = quote_plus(password)
+    
+    # MongoDB URI construction
+    uri = f'mongodb+srv://{encoded_username}:{encoded_password}@discountmatecluster.u80y7ta.mongodb.net/?retryWrites=true&w=majority&appName=DiscountMateCluster'
```
Comment (rashannon-sys) [inline Scrapping/Foodland Scraper/scraper_foodland.py:53]: Please remove the username, Thanks!

**Review by rashannon-sys — CHANGES_REQUESTED** [review]
Please remove the username, Thanks!

**Review by rashannon-sys — APPROVED** [review]

**Pull request merged**